### ISSUE TITLE
Error during printString in debugger (inspector) makes debugger unusable

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#49]
Errors should be tolerated in the inspector and anywhere else where messages are being sent to developer objects as often the problem being debugged may involve the printString:
```
a OffsetError occurred (error 2003), reason:objErrBadOffsetIncomplete, max:0 actual:1
---
OffsetError (AbstractException) >> _signalWith: @5 line 25
OffsetError (AbstractException) >> signal @2 line 47
UUID (Object) >> _error:args: @15 line 11
UUID (Object) >> _errorIndexOutOfRange: @2 line 6
UUID (ByteArray) >> at: @4 line 13
UUID >> printHexAt:to: @3 line 4
UUID >> asString @7 line 5
UUID >> printString @2 line 3
RowanFrameService >> varsFor: @62 line 33
RowanFrameService >> initializeProcess:level:organizer: @18 line 13
RowanFrameService class >> process:level:organizer: @3 line 4
RowanProcessService >> initialize:status: @9 line 7
RowanProcessService class >> onActiveProcess: @3 line 4
RowanDebuggerService >> update @3 line 3
[] in  ExecBlock1 (JadeServer) >> updateFromSton: @5 line 10
Array (Collection) >> do: @5 line 10
JadeServer64bit32 (JadeServer) >> updateFromSton: @7 line 6
GsNMethod class >> _gsReturnToC @1 line 1
```

**Comment by jgfoster** [comment]
I have a fix waiting on #60.

**Comment by ericwinger** [comment]
Merged Jadeite & Rowan updates and new test #testErrorInPrintString passes when run individually. Reassigning back to @dalehenrich for verification of fix after Jadeite 3.0.12 is release. 

@jgfoster I'm still seeing debugger test hangs when all debugger tests are run together.

**Comment by LisaAlmarode** [comment]
If I execute String foo and debug, the debugger opens with an error dialog that starts with "Error when attempting to read stack frame. Maybe an object got an error on a #printString method.... a reference using non-existent key #'rowanCommandResults was made into... aSessionTemps.  

While I'm not sure the exact nature of the bug, I'll hazard a guess it's not entirely fixed.

**Comment by jgfoster** [comment]
@LisaAlmarode, 
I just tried this with Rowan 490479e4d18f56fef7b4cc38c422530f2369257e and Jadeite 815805e1095c23ceab32e9ac2e7a523a9086d8ed. Following your instructions, I typed 'String foo' in a workspace, clicked 'Debug' in the resulting dialog, and observed a nice debugger appear. I'll need more details on the versions involved and confirmation that it reproduces on a build with my changes.

<img width="448" alt="screen shot 2018-08-17 at 7 04 50 pm" src="https://user-images.githubusercontent.com/1577872/44294707-89ee9d80-a250-11e8-84db-db2ae6554fbf.png">

<img width="689" alt="screen shot 2018-08-17 at 7 05 23 pm" src="https://user-images.githubusercontent.com/1577872/44294709-8c50f780-a250-11e8-9a1c-835b1d9a7d33.png">

**Comment by LisaAlmarode** [comment]
Tried again with latest rowan 292f145, same oscar 3.0.12, and I don't get the String foo issue. 

If I try the right pop up menu in the code pane, it throws up an error dialog (this is reported issue #64, comment 2), and after that various other alarming error dialogs appear, the first one is this, but afterwards XML parsing errors and DNUs
![image](https://user-images.githubusercontent.com/13142166/44302669-b3580980-a2e2-11e8-98a5-d3038327612f.png)

Will close this and retest when I have fix for 64, and if necessary file separate issue.
